server/routes/index.coffee

```diff
@@ -59,6 +59,7 @@ module.exports.setup = (app) ->
   Course = require '../models/Course'
   app.get('/db/course', mw.rest.get(Course))
   app.get('/db/course/:handle', mw.rest.getByHandle(Course))
+  app.get('/db/course/:handle/levels/:levelOriginal/next', mw.courses.fetchNextLevel) # For playing in teacher mode, which doesn't have a courseInstance
```

Comment:
Probably shouldn't have comments like this. It's not the server's concern, and it's likely to get out of date. The process for finding a route's current purpose should be

1. Find the Model or Collection with the function that interfaces with the endpoint
1. Find instances of the function being called.